OpenEdXMobile/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -266,7 +266,8 @@ public void showErrorDialog(String header, String message) {
     }
 
     @Override
-    public boolean createOptionsMenu(Menu menu) {
+    protected void onResume() {
+        super.onResume();
```

Comment:
I suppose the right thing to do is to set these default display option in the theme instead of dynamically, where they would be picked up by the Toolbar on initialization automatically, thus removing the need for duplicated explicit initialization in every Activity as well as a side bonus. The rest of your changes look good!